using-mobile-extensions/adobe-media-analytics/migration.md

```diff
@@ -0,0 +1,947 @@
+# Migrating to AEPMedia reference
+
+This document is a reference comparison of AEPMedia \(3.x\) APIs against their equivalent ACPMedia \(2.x\) APIs.
+
+The AEPMedia extension is implemented purely in Swift and is compatible with the AEPCore swift SDK. To ensure a smooth transition from the ACPMedia SDK, there are no major changes on the API names or definition. For more details, follow the migration guide below for your Swift or Objective-C mobile application.  If explanation beyond showing API differences is necessary, it will be captured as an info hint within that API's section.
+
+## Public classes
+
+| Type          | AEP 3.x (Swift) | AEP 3.x (Objective-C) | ACP 2.x (Objective-C) |
+| ------------- | :-------------- | --------------------- | --------------------- |
+| Primary Class | Media           | AEPMobileMedia        | ACPMedia              |
+
+
+
+## Public APIs
```

Comment:
Same comment here. It feels a little weird to say "Public APIs" here when, by definition, these docs are public. I'd go with "AEPMedia APIs", or something like that instead.